commcare_connect/templates/pagination.html

```diff
@@ -16,21 +16,67 @@
         </li>
       {% endif %}
 
-      {% for page_number in page_obj.paginator.page_range %}
```

Comment:
I had to make this change because the previous pagination template displayed all the page numbers, and as the number of pages increased, it started overflowing.